zerver/lib/users.py

```diff
@@ -204,13 +204,14 @@ def bulk_get_cross_realm_bots() -> Dict[str, UserProfile]:
     return {user.email.lower(): user for user in users}
 
 
-def user_ids_to_users(user_ids: Sequence[int], realm: Realm) -> List[UserProfile]:
-    # TODO: Consider adding a flag to control whether deactivated
-    # users should be included.
-
-    user_profiles = list(
-        UserProfile.objects.filter(id__in=user_ids, realm=realm).select_related("realm")
-    )
+def user_ids_to_users(
+    user_ids: Sequence[int], realm: Realm, *, allow_deactivated: bool = True
```

Comment:
Not sure of the default value here. Currently `allow_deactivated` is always passed as `False`, but added the flag just for future use and as it was mentioned as TODO.